pkg/worker/pgq/pgq_processor.go

```diff
@@ -108,6 +108,8 @@ func (p *Processor) Process(ctx context.Context, types []string, fn worker.JobEx
 			return fmt.Errorf("pickup job: %w", err)
 		}
 
+		// check for dupes, only for sync asset jobs
```

Comment:
Is there something to be added here?

Reply:
nope, removed this comment. Validation already there after querying the jobs_queue